### PR TITLE
add a changelog for LAPACKFullMatrix::solve()

### PR DESCRIPTION
**Pull request by davydden**
a changelog for https://github.com/dealii/dealii/pull/5585

### CODE REVIEW

File: doc/news/changes/minor/20171207DenisDavydov [hunk 1]
```diff
@@ -0,0 +1,5 @@
+New: Improve support of triangular matrices in LAPACKFullMatrix wrappers.
+Add LAPACKFullMatrix::solve() to solve a system of equations either when
```
Comment (kronbichler) [inline doc/news/changes/minor/20171207DenisDavydov]: Maybe say "Improve support of triangular matrices in LAPACKFullMatrix wrappers by adding a LAPACKFullMatrix::solve() functino that solves ..." to make the style more uniform with the other changelog entries...

Comment (davydden) [reply doc/news/changes/minor/20171207DenisDavydov]: actually that `solve()` is NOT only for triangular, it now uses Cholesky also

Comment (davydden) [reply doc/news/changes/minor/20171207DenisDavydov]: so it is related, but it not limited to improvements related to triangular matrices.

**Comment by bangerth** [comment]
It's just a changelog entry. Let's not agonize over this :-)

**Pull request merged**